py-polars/debug/launch.py

```diff
@@ -0,0 +1,51 @@
+import re
+import subprocess
+import sys
+import time
+from pathlib import Path
+
+
+def launch_debugging() -> None:
```

Comment:
What happens when I do the vscode button for run without debugging. Right now, its just giving me this:
![image](https://github.com/pola-rs/polars/assets/37670616/61b8cf7f-dc96-41e4-9508-f9d87124c4db)

Reply:
`launch.py` isn't meant to be called directly. But I just pushed a commit that will raise an exception with a message not to run it directly.

Reply:
FYI "run without debugging" is a VSCode feature that I think isn't really well-implemented in the Python extension. I've never actually used it.